src/main/kotlin/com/shortener/exception/ExceptionHandler.kt

```diff
@@ -0,0 +1,23 @@
+package com.shortener.exception
+
+import org.springframework.core.Ordered
+import org.springframework.core.annotation.Order
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.servlet.ModelAndView
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler
+import javax.servlet.http.HttpServletRequest
+
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@ControllerAdvice
+class ExceptionHandler : ResponseEntityExceptionHandler() {
+
+    @ExceptionHandler(Exception::class, InvalidInputUrl::class)
+    fun handleError(req: HttpServletRequest, ex: Exception): ModelAndView? {
```

Comment:
when is this function returning null so that the ModelAndView is nullable?

Reply:
Well, never. Didn't know what I was doing